src/components/AppBar.tsx

```diff
@@ -82,7 +82,10 @@ function SmallWindowNav() {
   )
 }
 
-function Dropdown({ children, target }: React.PropsWithChildren<{ target: React.ReactElement }>) {
+export function Dropdown({
```

Comment:
Good thinking that we can reuse the `Dropdown` component! 

Now that it is exported and used in multiple places, it should probably live in its own file at `src/components/dropdown.tsx` and reexported from `src/components/index.tsx`.